src/pages/programDisaggregations/form/ProgramIndicatorMappingSection.tsx

```diff
@@ -25,7 +28,9 @@ export const ProgramIndicatorMappingSection = ({
     initialProgramIndicators: ProgramIndicatorWithMapping[]
```

Comment:
I think this can be removed and instead use 
```ts 
const { initialValues } = useFormState<ProgramDisaggregationFormValues>({
        subscription: { initialValues: true },
    })`